docs/guides/deployment/deploy-to-fly-io-with-sqlite.md

```diff
@@ -0,0 +1,104 @@
+# Deploy to Fly.io with SQLite
+
+:::note
+
+To follow this how-to guide, you'll first need to install the Fly CLI and create
+an account by [following this official guide](https://fly.io/docs/hands-on/).
+
+:::note
+
+Navigate to your Platformatic DB project in the terminal on your local machine.
+Run `fly launch` and follow the prompts. When it asks if you want to deploy
+now, say "no" as there are a few things that you'll need to configure first.
+
+<!-- TODO: Could probably put a command here to launch the app in non-interactive mode -->
```

Comment:
Yep, if we want a one-liner:

```sh
fly launch --no-deploy --generate-name --region lhr --org personal --path .
```

The one-liner makes a number of assumptions but these seem like sane defaults:
* a random name is ok
* a geographically central instance is ok ([region list is in the docs](https://fly.io/docs/reference/regions/#fly-io-regions))
* user is deploying to their own organisation